frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/LoginHandler.java

```diff
@@ -178,24 +178,48 @@ public APIGatewayProxyResponseEvent handleRequestWithUserContext(
                 } else {
                     codeStorageService.increaseIncorrectPasswordCount(request.getEmail());
                 }
-
-                auditService.submitAuditEvent(
-                        FrontendAuditableEvent.INVALID_CREDENTIALS,
-                        userContext.getClientSessionId(),
-                        userContext.getSession().getSessionId(),
-                        clientId,
-                        internalCommonSubjectIdentifier.getValue(),
-                        request.getEmail(),
-                        IpAddressHelper.extractIpAddress(input),
-                        AuditService.UNKNOWN,
-                        persistentSessionId,
-                        pair("internalSubjectId", userProfile.getSubjectID()),
-                        pair(
-                                "incorrectPasswordCount",
-                                codeStorageService.getIncorrectPasswordCount(request.getEmail())),
-                        pair("attemptNoFailedAt", configurationService.getMaxPasswordRetries()));
-
-                return generateApiGatewayProxyErrorResponse(401, ErrorResponse.ERROR_1008);
+                incorrectPasswordCount++;
+                if (incorrectPasswordCount >= configurationService.getMaxPasswordRetries()) {
+                    LOG.info("User has now exceeded max password retries");
+
+                    auditService.submitAuditEvent(
+                            FrontendAuditableEvent.ACCOUNT_TEMPORARILY_LOCKED,
+                            userContext.getClientSessionId(),
+                            userContext.getSession().getSessionId(),
+                            clientId,
+                            internalCommonSubjectIdentifier.getValue(),
+                            userProfile.getEmail(),
+                            IpAddressHelper.extractIpAddress(input),
+                            userProfile.getPhoneNumber(),
+                            persistentSessionId,
+                            pair("internalSubjectId", userProfile.getSubjectID()),
+                            pair(
+                                    "attemptNoFailedAt",
+                                    configurationService.getMaxPasswordRetries()));
+
+                    return generateApiGatewayProxyErrorResponse(400, ErrorResponse.ERROR_1028);
+                } else {
+                    auditService.submitAuditEvent(
```

Comment:
should we send an INVALID_CREDENTIALS event in all cases, even in addition to ACCOUNT_TEMPORARILY_LOCKED to record the number of times that the credentials were entered incorrectly?